app/src/main/java/com/android/fundamentals/workshop04/WS04ActorsAdapter.kt

```diff
@@ -0,0 +1,61 @@
+package com.android.fundamentals.workshop04
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.android.fundamentals.R
+import com.android.fundamentals.data.models.Actor
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+
+class WS04ActorsAdapter : RecyclerView.Adapter<DataViewHolder>() {
+
+    private val imageOption = RequestOptions()
+        .placeholder(R.drawable.ic_avatar_placeholder)
+        .fallback(R.drawable.ic_avatar_placeholder)
+        .circleCrop()
+
+    private var actors = mutableListOf<Actor>()
```

Comment:
Mutability is not used

Reply:
Or make it `val` and operate with a mutable list, or keep it var but make the list immutable